*** nu.validator.htmlparser.HtmlParser.nocache.js~	Wed May  5 14:56:28 2010
--- nu.validator.htmlparser.HtmlParser.nocache.js	Wed May  5 15:12:56 2010
***************
*** 2825,2831 ****
--- 2825,2833 ----
    $processNonNcNames(attributes, this$static, this$static.namePolicy);
    elt = $createElement_0(this$static, ns, elementName.name_0, attributes);
    current = this$static.stack_0[this$static.currentPtr];
+   if (current) {
    current.fosterParenting?$insertIntoFosterParent(this$static, elt):$appendElement(this$static, elt, current.node);
+   }
    node = $StackNode_0(new StackNode, ns, elementName, elt);
    $push_0(this$static, node);
  }
